src/http_accessor_requests.py

```diff
@@ -34,6 +34,8 @@ def access(self, data=None):
             return self._http_post(data=data)
         elif self.method == 'PATCH':
             return self._http_patch(data=data)
+        elif self.method == 'PUT':
+            return self._http_put(data=data)
```

Comment:
do we have  _http_put ? looks like need to add one. Please help to have a test.

just set "use_curl = True" to be "use_curl = Frue" in OobUpdate.py, to use http_accessor_requests.

Reply:
Done, tested the code with "use_curl = True"